src/filtering.rs

```diff
@@ -143,6 +228,7 @@ impl SnapshotFilter {
     /// `true` if the snapshot matches the filter, `false` otherwise
     #[must_use]
     pub fn matches(&self, snapshot: &SnapshotFile) -> bool {
+        #[cfg(feature = "rhai")]
```

Comment:
See discord, maybe it makes sense to make jq and rhai filter options mutually exclusive?
Or what is the intended way that both interact with each other when both are set?